res/xml/preferences.xml

```diff
@@ -164,6 +164,10 @@
 				android:defaultValue="false"
 				android:title="@string/settings_auth_full_uri"
 				android:summary="@string/settings_auth_full_uri_info"/>
+		<CheckBoxPreference
+			android:key="publish_enable"

```

Comment:
Generally speaking, this is a per-account setting. Now, after working on a UA for many years, the number of times I have toggled that setting amount to a grand total of 0. (other than testing, of course).

So, it's probably not bad to have it as a global setting.